source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/organization-select/display.tsx

```diff
@@ -36,6 +37,7 @@ class Display extends React.Component<IProps> {
       options: organizationOptions,
       onChange: this.onChange,
       value: defaultValue,
+      icon: <ArrowDropDownIcon />,
```

Comment:
this icon not existing isn't directly related to this set of changes, but the icon was missing, and I figured I might as well fix it while I'm in here